samples/nrf9160/modem_shell/overlay-location_service_ext.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Modem Shell
+CONFIG_MOSH_CLOUD_MQTT=y
+CONFIG_MOSH_CLOUD_REST=n
```

Comment:
We could use `overlay-cloud_mqtt.conf` instead of adding these here.
So we could have either current solution:
`west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG="overlay-location_service_ext.conf"`
or
`west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG="overlay-cloud_mqtt.conf" -DCONFIG_LOCATION_SERVICE_EXTERNAL=y`

Opinions?

Reply:
I'm fine with the latter approach as well. This feature is mostly for our own testing, I guess, so I'm not sure if we really need to have an overlay.

Reply:
I'm removing this overlay so we'll go with:
`west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG="overlay-cloud_mqtt.conf" -DCONFIG_LOCATION_SERVICE_EXTERNAL=y`